src/Mono.Android/Android.Views/View.cs

```diff
@@ -91,7 +89,7 @@ public bool FitsSystemWindows ()
 		}
 #endif
 
-#if NET && ANDROID_34
+#if ANDROID_34
```

Comment:
I think `ANDROID_34` could be deleted as well. We won't be making a `net8.0-android34` or `net9.0-android34`, etc.

Reply:
Maybe we will want to go through and clean this up in a future PR? Given that main still allows you to build Mono.Android.csproj for ~any supported framework version I think it makes sense to leave these in for now.